docs/changes/utils/13661.api.rst

```diff
@@ -0,0 +1 @@
+``astropy.utils.misc.possible_filename`` has been removed.
```

Comment:
@eerovaher said we don't need a change log?

Personally, I don't think it hurts to have. No one ever goes back to old change logs and then complain "hey but you never mention this before". But then again, you said this is "private" so maybe we don't need to mention... I'll leave it to your good judgement.

Reply:
Agreed.